addons/print_docsaway/models/print_docsaway.py

```diff
@@ -0,0 +1,391 @@
+import json
+import urllib2
+
+from ssl import SSLError
+from openerp import api, fields, models
+from openerp.exceptions import Warning
+from openerp.addons.print_docsaway.data.docsaway_error import _DOCSAWAY_ERROR_ACCOUNT, _DOCSAWAY_ERROR_STATION, _DOCSAWAY_ERROR_PRICE, _DOCSAWAY_ERROR_MAIL
+from openerp.tools.translate import _
+
+
+# URLs for requests
+_DOCSAWAY_GETACCOUNT_URL = 'https://www.docsaway.com/app/api/rest/account.json'
```

Comment:
Shouldn't those variables be DocsAwayProvider class variables, to avoid exposing them too much ?